src/bumerang/app.py

```diff
@@ -35,6 +37,9 @@ def __init__(self):
             (r'/profile/([0-9]+)/?', ProfileHandler),
             (r'/request/?', BorrowHandler),
             (r'/request/([0-9]+)/?', BorrowHandler),
+            (r'/requests/recent/?', BorrowsByRecentHandler),
```

Comment:
Nah, the two lines is fine vs putting a `?` after the number, either works! I was just confused